en/reports/vivliostyle-report-2020/vf2020report.md

```diff
@@ -0,0 +1,263 @@
+
+- [theme-slide (Slide)](https://github.com/vivliostyle/themes/tree/master/packages/@vivliostyle/theme-slide)
+- [theme-bunko (Vertical Book)](https://github.com/vivliostyle/themes/tree/master/packages/@vivliostyle/theme-bunko)
+- [theme-techbook (Tech book)](https://github.com/vivliostyle/themes/tree/master/packages/%40vivliostyle/theme-techbook)
+
+#### [Create Book](https://github.com/vivliostyle/create-book)
+
+This is an installer that builds an environment for book production using the Vivliostyle CLI locally, and it is also a local version of the Vivliostyle Pub. It was created by [uetchy](https://github.com/uetchy) in the wake of the Mitou Advanced program defeat, which will be described later. As of the end of this fiscal year, v0.1.6 has been released. For details, please refer to the following article posted on the official website. 
+
+- [Feature Articles: Let's self-publish with Create Book!](https://vivliostyle.org/make-books-with-create-book/)
+
+#### [Vivliostyle Pub](https://github.com/vivliostyle/vivliostyle-pub)
+
+The Vivliostyle Pub is the culmination of our products. I will explain the development process in some detail in this section. The starting point is “Viola”, an online editor that runs in a browser. The author, [spring-raining](https://github.com/spring-raining), offered to give it to our organization so that the contributors could work together to recreate it. Later, during the monthly developer's meeting, the project was given the name "Vivliostyle Pub" by Representative Director Murakami, and development was officially started. This was also mentioned in the previous year's activity report.
+
+
+As of the end of the previous fiscal year, the biggest concern was securing development funding, and the solution was discussed at the developer meeting. Therefore, the contributors suggested that they apply for the [Mitou Advanced Project](https://www.ipa.go.jp/jinzai/advanced/2020/koubo_index.html) with the theme of developing Vivliostyle Pub. 
+
+As a result of discussions among the members, it was decided to take on the challenge. Since organizations are not allowed to apply due to regulations, we completed the application on April 2 as a group of individuals consisting of [youchan](https://github.com/youchan) and [takanakahiko](https://github.com/takanakahiko), in addition to the aforementioned contributors and Representative Director Murakami.
+
+From that point on, all of the members of the team worked together, and day after day, night after night, the development of Vivliostyle Pub began. The goal is the second round of review scheduled for mid-May. The following is an overview of the version (alpha version) at this point.
+
+
+- The Vivliostyle CLI deployed in the cloud implements a very early parser for VFM.
+- Vivliostyle CLI can output the result of CSS typesetting to the browser.
+- Vivliostyle CLI can generate and download a PDF of the CSS typesetting result from the Export menu.
```

Comment:
「Vivliostyle CLI はCSS組版の結果をブラウザに出力することができます。」
「Vivliostyle CLIは、ExportメニューからCSS組版結果のPDFを生成してダウンロードすることができます。」
とありますが、どちらもVivliostyle CLIが主語になっているのは間違いです。CLIが組版結果をブラウザに出力するわけではないし、CLIが組版結果をダウンロードするわけでありません。

修正案：

- It can output the result of CSS typesetting to the browser.
- The generated PDF can be downloaded from the Export menu.